script/setup/install-dev-tools

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env bash
 
+GOLANGCI_LINT_VERSION="v1.27.0"
+
 #
 # Install developer tools to $GOBIN (or $GOPATH/bin if unset)
 #
 set -eu -o pipefail
 
+# Enable Go modules
+export GO111MODULE=on
+
 # prevent updating go.mod of the project
 cd /tmp
-go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+go get "github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}"
```

Comment:
curious why the `-u` was removed

Also (see my other comment) might be better to inline the `GO111MODULE`;

```suggestion
GO111MODULE=on go get "github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}"
```